.swiftpm/xcode/xcshareddata/xcschemes/ResilientDecoding.xcscheme

```diff
@@ -41,10 +41,21 @@
       selectedDebuggerIdentifier = "Xcode.DebuggerFoundation.Debugger.LLDB"
       selectedLauncherIdentifier = "Xcode.DebuggerFoundation.Launcher.LLDB"
       shouldUseLaunchSchemeArgsEnv = "YES"
-      codeCoverageEnabled = "YES">
+      codeCoverageEnabled = "YES"
+      onlyGenerateCoverageForSpecifiedTargets = "YES">
+      <CodeCoverageTargets>
+         <BuildableReference
+            BuildableIdentifier = "primary"
+            BlueprintIdentifier = "ResilientDecoding"
+            BuildableName = "ResilientDecoding"
+            BlueprintName = "ResilientDecoding"
+            ReferencedContainer = "container:">
+         </BuildableReference>
+      </CodeCoverageTargets>
```

Comment:
Ensures code coverage numbers are accurate - previously was including the test target coverage too.